apps/service_providers/tracing/langfuse.py

```diff
@@ -213,3 +214,29 @@ def shutdown(self):
 def _shutdown():
     """Shutdown the client manager when the program exits."""
     client_manager.shutdown()
+
+
+class LangfuseCallbackHandler(CallbackHandler):
```

Comment:
I'm curious, if we do custom tracing, we'll be able to pick up this event as well?